domain-ee/ee-ep-merge-app/src/python_src/service/ep_merge_machine.py

```diff
@@ -48,8 +56,10 @@ class EpMergeMachine(StateMachine):
             | running_merge_contentions.to(completed_error, cond="has_error")
             | running_move_contentions_to_pending_claim.to(running_cancel_ep400_claim, unless="has_error")
             | running_move_contentions_to_pending_claim.to(completed_error, cond="has_error")
-            | running_cancel_ep400_claim.to(completed_success, unless="has_error")
+            | running_cancel_ep400_claim.to(running_add_claim_note_to_ep400, unless="has_error")
```

Comment:
I thought the plan was to add a claim note when merging failed, not on success?

Reply:
Nope. The plan is to add a claim note as part of the happy path. See #2161. The claim note is being added after the claim is cancelled that way the cancellation reason for the claim will show in VBMS. 👍